coml/core.py

```diff
@@ -78,6 +79,7 @@ def parse_code(response: str) -> str:
 class CoMLAgent:
     def __init__(self, llm: BaseChatModel):
         self.llm = llm
+        self.vis_verifier = VisVerifier(llm, self)
```

Comment:
This doesn't need to be a member variable as it's only used in `vis_check`.

It can be created when `vis_check` is called.